device/bgp/bgpsrtev6remoteendpointsubtlv.yaml

```diff
@@ -0,0 +1,31 @@
+components:
+  schemas:
+    BgpSrte.RemoteEndpointSubTlv:
```

Comment:
if this has both v4 and v6 ip then what is the point of having v4 and v6 tunnel files.

Reply:
we discussed this.